docs/usage/upload_files.rst

```diff
@@ -0,0 +1,199 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+Creating Buckets
+----------------
+
+In order to upload, modify or delete files, we first need to create a bucket.
```

Comment:
https://docs.aws.amazon.com/AmazonS3/latest/dev/Introduction.html
https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingBucket.html

A bucket is a container for objects. Every object is contained in a bucket.
Buckets have a default location (link to the related section) and storage class (link to the related section). 

Each bucket can also define a quota. The size of a bucket is the size of all objects in the bucket (including all versions).